### PR TITLE
Language handling added

### PR DESCRIPTION
**Pull request by mzhk**

### CODE REVIEW

File: crnk-example-frontend/src/app/common/language/common.language.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		else {
+			const browserLang = this.translate.getBrowserLang();
+			const languageToSelect = this.langs.includes(browserLang as LanguageCode) ? browserLang : this.defaultLang;
+			return languageToSelect as LanguageCode;
+		}
+	}
+
+	changeLanguage(language: LanguageCode) {
+		localStorage.setItem('language', language);
+		this.translate.use(language);
```
Comment (remmeier) [inline crnk-example-frontend/src/app/common/language/common.language.ts]: those two elements (setting local storage and updating translate could go into an ngrx effect? then anybody can just trigger the setLanguageAction, but it does only set the store.

Comment (remmeier) [reply crnk-example-frontend/src/app/common/language/common.language.ts]: getLanguageFromBrowser could potentially be put into an APP_INITIALIZER and then trigger the setLanguage action on startup? or maybe the effect initializes the defaults like here.

File: crnk-example-frontend/src/app/app.module.ts [hunk 2]
```diff
@@ -132,6 +134,11 @@ export const reducers: ActionReducerMap<any> = {};
 			useFactory: load_login,
 			deps: [LoginService],
 			multi: true
+		},
+		{
+			provide: HTTP_INTERCEPTORS,
```
Comment (remmeier) [inline crnk-example-frontend/src/app/app.module.ts]: maybe define in AppCommonModule?

Comment (mzhk) [reply crnk-example-frontend/src/app/app.module.ts]: Sounds good!

File: crnk-example-frontend/src/app/store/store.reducers.ts [hunk 3]
```diff
@@ -32,10 +32,24 @@ export function appReducer(
...
+		case AppActionTypes.SET_LANGUAGE: {
+			const action = actionObj as SetLanguageAction;
+			if (action.language) {
+				return {
+					...state,
+					language: action.language
+				};
+			}
+			else {
+				return {
```
[the remark this answers is not in the document]
Comment (mzhk) [reply crnk-example-frontend/src/app/store/store.reducers.ts]: ops ;)

**Pull request merged**